### PR TITLE
fix(Autosuggest): fix showing suggestions on focus input

### PR DESCRIPTION
**Pull request by mukiienko**
Fixed issue with loading expansions for related selected terms while input empty.
Task: https://jira.textkernel.nl/browse/ONEUI-234

# Checklist
- [x] The implementation has been manually tested and complies with Textkernel [browser support guidelines](https://textkernel.com/browser-support/)
- [x] The implementation complies with [accessibility](CONTIRBUTING.md#accessibility) standards.
- [x] The component has a [displayName](CONTRIBUTING.md#display-names) defined.
- [x] The component comes with a detailed [PropTypes](CONTRIBUTING.md#component-props) (and defaultProps) definition.
- [x] Component PropTypes are sufficiently described / documented.
- [x] There is [a story](CONTRIBUTING.md#component-showcases) in Storybook.

### CODE REVIEW

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js [hunk 1]
```diff
@@ -124,6 +131,16 @@ describe('Autosuggest', () => {
             expect(wrapper.find('input')).toHaveLength(1);
             expect(wrapper.find('.Autosuggest__input--hidden')).toHaveLength(1);
         });
+        it('should show empty suggestion placeholder while results are empty and input has value', () => {
+            wrapper.setProps({ noSuggestionsPlaceholder: 'no suggestions' });
+            setFocusOnInput();
+            expect(toJson(wrapper)).toMatchSnapshot();
```
Comment (eszthoff) [inline src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js:137]: I find snapshots are very hard to understand and very easy to mess up if something changes (just update them and the test passes). Could you add some assertions of what in the snapshot are we looking for?

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js [hunk 2]
```diff
@@ -53,6 +53,13 @@ describe('Autosuggest', () => {
             expect(toJson(wrapper)).toMatchSnapshot();
             expect(wrapper.find('ListItem')).toHaveLength(0);
         });
+        it('should initially render focused component with suggestions list correctly', () => {
+            suggestionsList = SUGGESTIONS.slice(1, 20);
+            wrapper.setProps({ suggestions: suggestionsList });
+            setFocusOnInput();
+            expect(toJson(wrapper)).toMatchSnapshot();
+            expect(wrapper.find('ListItem')).toHaveLength(8);
```
Comment (eszthoff) [inline src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js:61]: why 8?

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js [hunk 3]
```diff
@@ -124,6 +131,16 @@ describe('Autosuggest', () => {
             expect(wrapper.find('input')).toHaveLength(1);
             expect(wrapper.find('.Autosuggest__input--hidden')).toHaveLength(1);
         });
+        it('should show empty suggestion placeholder while results are empty and input has value', () => {
```
Comment (eszthoff) [inline src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js:134]: I think description is confusing a bit. Maybe say `should show "no suggestions" placeholder only if input has value"

**Review by eszthoff — CHANGES_REQUESTED** [review]

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js (hunk 2 above)
Comment (mukiienko) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js:61]: We pass the list of suggestions in that way:
```
suggestionsList = SUGGESTIONS.slice(1, 9);
```
But I agree, it is not obvious. I changed:
```
suggestionsList = SUGGESTIONS.slice(0, 8);
```

**Review by eszthoff — APPROVED** [review]

**Comment by carlobernardini** [comment]
@alexey the scope for your PR should probably be `Autosuggest` rather than `autocomplete`.

**Pull request merged**